examples/user_interfaces/toolmanager.py

```diff
@@ -83,10 +114,21 @@ def set_lines_visibility(self, state):
 fig.canvas.manager.toolbar.add_tool('zoom', 'foo')
 
 # Remove the forward button
-fig.canvas.manager.toolmanager.remove_tool('forward')
+tool_mgr.remove_tool('forward')
 
 # To add a custom tool to the toolbar at specific location inside
 # the navigation group
 fig.canvas.manager.toolbar.add_tool('Hide', 'navigation', 1)
 
+for i, c in enumerate(['yellowgreen', 'forestgreen']):
+    sidebar = fig.canvas.manager._get_toolbar()
```

Comment:
yes, I didn't like doing that either, part of the rounding I spoke of, wanting a second opinion.

Do you know why we have ``_get_toolbar`` as an internal method?  Also the name seems a little weird as it doesn't get the toolbar, but it creates one.

I could also go with something like ``fig.canvas.manager.get_backend.Toolbar()``.  What do you think?